sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -14,25 +15,43 @@
     private final String text;
     private final SentimentConfidenceScores confidenceScores;
     private final TextSentiment sentiment;
+    private final IterableStream<MinedOpinions> minedOpinions;
 
     /**
      * Creates a {@link SentenceSentiment} model that describes the sentiment analysis of sentence.
-     *
-     * @param text The sentence text
+     * @param text The sentence text.
      * @param sentiment The sentiment label of the sentence.
      * @param confidenceScores The sentiment confidence score (Softmax score) between 0 and 1, for each sentiment label.
      *   Higher values signify higher confidence.
      */
     public SentenceSentiment(String text, TextSentiment sentiment, SentimentConfidenceScores confidenceScores) {
         this.text = text;
         this.sentiment = sentiment;
+        this.minedOpinions = null;
```

Comment:
This is final, does it mean it is always null on this model class?

Reply:
if user uses this constructor, that means the the minedOpinion will always be null.
null minedOpinon means user doesn't want to include opinion mining in the request.
empty list of minedOpinion has different meaning. It means the user wants the opinion mining but have no opinion returns, so it is an empty list of opinions.